src/cmds/hardware/lsblk/lsblk.c

```diff
@@ -5,17 +5,56 @@
  * @date Anton Bondarev
  */
 #include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
 #include <inttypes.h>
+#include <getopt.h>
 
 #include <drivers/block_dev.h>
 
+static const char *convert_unit(uint64_t *size) {
+	const char *unit;
+	if (*size < 1024) {
+		unit = "B";
+	}
+	else if (*size < 1024 * 1024) {
+		unit = "KB";
+		*size = *size/1024.0;
+	}
+	else if (*size < 1024ull * 1024 * 1024) {
+		unit = "MB";
+		*size = *size/(1024.0 * 1024);
+	}
+	else if (*size < 1024ull * 1024 * 1024 * 1024) {
+		unit = "GB";
+		*size = *size/(1024.0 * 1024 * 1024);
+	} else {
+		unit = "TB";
+		*size = *size/(1024.0 * 1024 * 1024 * 1024);
+	}
+	return unit;
+}
+
 int main(int argc, char **argv) {
 	int i;
 	struct block_dev **bdevs;
 
 	bdevs = get_bdev_tab();
 	assert(bdevs);
 
+	int c;
```

Comment:
code-style.
Please move variable declaration to the start of the block (to the declaration part)